node/network/statement-distribution/src/lib.rs

```diff
@@ -1542,6 +1549,16 @@ impl StatementDistribution {
 		// Sender/Receiver for getting news from our responder task.
 		let (res_sender, mut res_receiver) = mpsc::channel(1);
 
+		ctx.spawn(
```

Comment:
Iiuc, this is done to avoid introducing backpressure for now, which we will eventually want. Correct me if I am wrong.

Reply:
No, we spawn a task handling incoming requests and it will back pressure on those. We spawn a task, so we can back pressure incoming requests without blocking the whole subsystem.